.gazelcfg.json

```diff
@@ -0,0 +1,6 @@
+{
+	"GoPrefix": "k8s.io/test-infra",
+	"SkippedPaths": [
+		"jenkins/aws-janitor"
```

Comment:
@zmerlynn didn't check in vendored code so it won't build in a sandbox. I can add the vendored code if we'd like, but for now I didn't want to touch it.